src/app/server/CommissioningWindowManager.cpp

```diff
@@ -51,12 +51,24 @@ void CommissioningWindowManager::OnPlatformEvent(const DeviceLayer::ChipDeviceEv
     if (event->Type == DeviceLayer::DeviceEventType::kCommissioningComplete)
     {
         ChipLogProgress(AppServer, "Commissioning completed successfully");
+        DeviceLayer::SystemLayer().CancelTimer(HandleCommissioningWindowTimeout, this);
+        mCommissioningTimeoutTimerArmed = false;
         Cleanup();
+        mServer->GetSecureSessionManager().ExpireAllPairings(kUndefinedNodeId, kUndefinedFabricIndex);
```

Comment:
Still reading through the rest of this, but a question here: is this trying to expire all PASE sessions?  If so, this doesn't seem like it would work right, since a post-AddNOC PASE session would not have kUndefinedFabricIndex as the fabric index.

We should probably just add an ExpireAllPASESessions on session manager.  We used to not be able to ask a session whether it's a PASE session, but now we can, via `GetSecureSessionType`.

Reply:
Will change - that's better and exactly what I was attempting to do.

Reply:
Added in the next commit